web-admin/src/main/java/pl/edu/icm/unity/webadmin/attributetype/ImportAttributeTypeDialog.java

```diff
@@ -0,0 +1,258 @@
+/*
```

Comment:
General remark: please split this class into two. One (likely in engine, with some API, maybe even in one of existing AttributeType related classes - to load. This one should have the UI logic - its is UI dialog.